packages/mco/components/disaster-recovery/application-dr-status/dr-status-table.tsx

```diff
@@ -0,0 +1,226 @@
+import * as React from 'react';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { GreenCheckCircleIcon } from '@openshift-console/dynamic-plugin-sdk';
+import { DRPlacementControlKind } from 'packages/mco/types';
```

Comment:
```suggestion
import { DRPlacementControlKind } from 'packages/mco/types';
```
This import looks wrong to me.